README.md

```diff
@@ -1288,6 +1288,7 @@ Update Time, five active automations, webhooks.
   * [Astra](https://www.datastax.com/products/datastax-astra/) — Cloud Native Cassandra as a Service with [80GB free tier](https://www.datastax.com/products/datastax-astra/pricing)
   * [codehooks.io](https://codehooks.io/) — JavaScript serverless API/backend and database service with functions, Mongdb-ish queries, key/value lookups, a job system, and a message queue. One instance free per project, 5000 records, 5000 calls/month free, three developers included. No credit-card required.
   * [CrateDB](https://crate.io/) - Distributed Open Source SQL database for real-time analytics. [Free Tier CRFREE](https://crate.io/lp-crfree): One-node with 2 CPUs, 2 GiB of memory, 8 GiB of storage. One cluster per organization, no payment method needed.
+  * [FalkorDB](https://www.falkordb.com/) — Managed native Graph DBMS / analytics platform with a Cypher query language. Limits on graph size up to 100 MB.
```

Comment:
All I can see here is that its a preview - not details about what is free etc, can you share a link here that details the free offering?

Generally we dont take preview services.

Reply:
It's not a preview but a full free tier.

See https://app.falkordb.cloud/

Reply:
This link shows nothing about the terms of the offering and what is free.  I am not signing up to find out. /however tge "Start Free" page shows this and has the title "Try FalkorDB for Free", so I am going on what I can see 

<img width="562" alt="Screenshot 2024-06-20 at 15 49 23" src="https://github.com/ripienaar/free-for-dev/assets/82342/a6bdd989-d748-4ef9-aec1-260ce7fbaa86">